man/coap_attribute.txt.in

```diff
@@ -24,7 +24,12 @@ int _flags_);*
 *coap_attr_t *coap_find_attr(coap_resource_t *_resource_, const unsigned 
 char *_name_, size_t _nlen_);*
 
-Link with -lcoap-@LIBCOAP_API_VERSION@.
+Dynamically link with -lcoap-@LIBCOAP_API_VERSION@ 
+or -lcoap-@LIBCOAP_API_VERSION@-openssl
+
+Statically link with libcoap-@LIBCOAP_API_VERSION@.a,
```

Comment:
What speaks against `-lcoap-@LIBCOAP_API_VERSION@` for static linkage?
(This paragraph gives no indication how to link the application.)